docs/adr/0019-use-workspace-dependencies.md

```diff
@@ -0,0 +1,47 @@
+# Use workspace dependencies in package.json
+
+- Deciders: Danny Coates
+- Date: 2020-05-27
+
+## Context and Problem Statement
+
+Since becoming a monorepo we haven't found a way to share code across packages that works well in local development, CI, and production without fragile, ad-hoc workarounds. We've used a mix of the following strategies so far:
+
+#### Using "long" relative paths
```

Comment:
FWIW, I'd throw these at the end of the doc. IMO, ADRs are best when a quick summary of the decision comes at the top and further context comes later. Inverted pyramid structure and all that.